bank_management/utils/validation.go

```diff
@@ -36,11 +36,13 @@ func validateName(name string) (bool, string) {
 	}
 }
 
-// func validateWithDrawAmount(customerDetails bankOperation.AccountInfo, amount float64) (bool, string) {
-
-// 	if amount > 0 {
-// 		return true, ""
-// 	} else {
-// 		return false, "Enter correct amount"
-// 	}
-// }
+func validateWithDrawAmount(total float64,amount float64) (bool, string) {
```

Comment:
return error instead of string like 
err := errors.New("insufficient amount")
return false, err

Reply:
what to do when there is no error
return true, "" 
what to write in place of empty string